js/binance.js

```diff
@@ -775,6 +775,8 @@ module.exports = class binance extends Exchange {
     }
 
     async fetchDepositAddress (code, params = {}) {
+        await this.loadMarkets ();
+
```

Comment:
This empty lines breaks the transpiler, but I'll edit it upon merging. Thx!

Reply:
ah, thanks. Will lint next time!